### ISSUE TITLE
xalan-c: executable not added to PATH

### ISSUE THREAD

**Issue by truh** [NixOS/nixpkgs#18300]
## Issue description

The executable Xalan is not added to PATH with package installation. Executing Xalan is therefore not easily possible.

The expression found [here](https://github.com/NixOS/nixpkgs/blob/52b64cedec801770287eb0fa42263732bf2ab562/pkgs/development/libraries/xalanc/default.nix) adds a prefix `usr/` to each of the files in the package. Removing the prefix fixes this issue but breaks the package [libdigidocpp](https://github.com/NixOS/nixpkgs/blob/52b64cedec801770287eb0fa42263732bf2ab562/pkgs/development/libraries/libdigidocpp/default.nix).

Package is maintained by @jagajaga.
### Steps to reproduce

```
$ nix-shell -p xalanc
$ Xalan
bash: Xalan: command not found
```
## Technical details
- System: 16.09pre87999.20f009d (Flounder)
- Nix version: nix-env (Nix) 1.11.2
- Nixpkgs version: 16.09pre87999.20f009d

**Comment by garbas** [comment]
@truh I would suggest adding 

```nix
postInstall = ''
  ln -s $out/usr/bin $out/bin
'';
```

to xalanc expression

**Comment by tomberek** [comment]
create PR or close?

**Comment by SuperSandro2000** [comment]
fixed in 2e0dfb49f7a973e8701e00c88c4e97340fa6c623
